src/libopensc/card-mcrd.c

```diff
@@ -26,15 +26,15 @@
 #include "config.h"
 #endif
 
-#include <stdlib.h>
-#include <string.h>
-#include <ctype.h>
-
 #include "internal.h"
 #include "asn1.h"
 #include "cardctl.h"
 #include "gp.h"
 
+#include <ctype.h>
+#include <string.h>
+#include <stdlib.h>
+
```

Comment:
Why is this moved below? It is a good practice to have system includes on the top and local on the bottom, as it is in the rest of the code base.

Reply:
Reverted,  but I follow the C++ guidlines:
* corresponded header file
* necessary project headers
* 3rd party libraries headers
* standard libraries headers
* system headers